tests/e2e/helpers.go

```diff
@@ -1362,6 +1362,11 @@ func waitForGenericRemediationToBeAutoApplied(t *testing.T, f *framework.Framewo
 			return false, nil
 		}
 		E2ELogf(t, "Found remediation: %s\n", remName)
+		if rem.Status.ApplicationState == compv1alpha1.RemediationNotApplied || rem.Status.ApplicationState == compv1alpha1.RemediationPending {
+			E2ELogf(t, "Retrying. remediation not yet applied. Remediation Name: %s, ApplicationState: %s\n", remName, rem.Status.ApplicationState)
```

Comment:
Looks like we got a clean run - but I didn't see any of this output in the log. Must have been a lucky run?

Have you had other runs where this increased CI stability?

Reply:
I think what fixed the issue is the sleep function after this one